l5kit/l5kit/environment/envs/l5_env.py

```diff
@@ -0,0 +1,323 @@
+import random
+from collections import defaultdict
+from typing import Any, DefaultDict, Dict, List, NamedTuple, Optional
+
+import gym
+import numpy as np
+import torch
+from gym import spaces
+from gym.utils import seeding
+
+from l5kit.configs import load_config_data
+from l5kit.data import ChunkedDataset, LocalDataManager
+from l5kit.dataset import EgoDataset
+from l5kit.environment.kinematic_model import KinematicModel, UnicycleModel
+from l5kit.environment.reward import CLEReward, Reward
+from l5kit.environment.utils import ActionRescaleParams, calculate_rescale_params, convert_to_numpy
+from l5kit.rasterization import build_rasterizer
+from l5kit.simulation.dataset import SimulationConfig, SimulationDataset
+from l5kit.simulation.unroll import (ClosedLoopSimulator, ClosedLoopSimulatorModes, SimulationOutputCLE,
+                                     UnrollInputOutput)
+
+
+class SimulationConfigGym(SimulationConfig):
+    """Defines the default parameters used for the simulation of ego and agents around it in L5Kit Gym.
+
+    :param eps_length: the number of step to simulate per episode in the gym environment.
+    """
+
+    def __new__(cls, eps_length: int = 32, start_frame_idx: int = 0) -> 'SimulationConfigGym':
```

Comment:
I read online: [Ref](https://stackoverflow.com/questions/42385916/inheriting-from-a-namedtuple-base-class/42386174#42386174). It says we need to override __new__ in case of namedTuples.